### PR TITLE
Upgrade polymake to 4.9

### LINKED ISSUE #34866: Upgrade polymake to 4.9
<div id="comment:0"></div>

https://polymake.org/doku.php/news/release_4_8
https://polymake.org/doku.php/news/release_4_9

Component: **packages: optional**

Author: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34866_

### PR DESCRIPTION
**Pull request by mkoeppe**
### 📚 Description

Fixes #34866 

### 📝 Checklist

- [ ] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [x] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: build/pkgs/polymake/spkg-install.in [hunk 1]
```diff
@@ -21,6 +21,10 @@ if [ -x $SAGE_LOCAL/bin/lrs1 ]; then
     more_configure_options="$more_configure_options --with-lrs=$SAGE_LOCAL"
 fi
 
+# Put these includes on the front, to avoid shadowing by installed headers
+# from a previous version. (polymake 4.8 puts these includes to the end.)
+export CXXFLAGS="-I$(pwd)/include/core-wrappers -I$(pwd)/include/core $CXXFLAGS"
```
Comment (kliem) [inline build/pkgs/polymake/spkg-install.in:26]: How did you find out, that you need to do this?

**Comment by kliem** [comment]
It's wonderful how this works out of the box nowadays. Just typing `sage -i polymake` does the trick.

File: build/pkgs/polymake/spkg-install.in (hunk 1 above)
Comment (mkoeppe) [reply build/pkgs/polymake/spkg-install.in:26]: There was an error when I tried the 4.8 upgrade, and I stared long enough at the command lines...

Comment (kliem) [reply build/pkgs/polymake/spkg-install.in:26]: Ok, there really is no reference to this solution.

**Review by kliem — APPROVED** [review]

**Comment by mkoeppe** [comment]
Thank you!

**Pull request merged**